### ISSUE TITLE
Design assets and templates

### ISSUE THREAD

**Issue by tadanna** [Green-Software-Foundation/meetup#46]
Hey @Jenya-design , could you confirm if we have a template for GSF events covers? [Here](https://drive.google.com/drive/folders/1S0x3yi3pDlfE3vzTpJp9VCU-k2JQZyBN) I see only the cover for the groups. 

I can see in the MeetUp groups that there were some designs developed for the CarbonHack. Do we have something for a general event? Let me know in the comments, and if not we can update the issue to create this. 

FYI @holanita

**Comment by tadanna** [comment]
@holanita thanks. I believe these are for the Summit tho, should we clean this up and create new ones for general events?

**Comment by Jenya-design** [comment]
Hello @tadanna 
I believe there's no card for 'general event', like events made in the main GSF style 
Usually I'm asked to make the logo and the design assets for a specific events like Summit or CarbonHack

Currently I'm working to create the themes that we can reuse for the events, there will be some basic templates also, you can check last updates [this issue](https://github.com/Green-Software-Foundation/ops-content/issues/6) and add your feedback

Let me know if you need any specific assets or formats created
Thank you

cc @holanita

**Comment by tadanna** [comment]
agreed @holanita , but I believe we can already prepare 2 templates for the coming events:
1. in-person
2. online (even if right now we might not use it.)

It would be great to have them ready and saved, and whenever we have a confirmed event we only need to populate: date, location & host(s). 

I like the idea to have 2 different colors - I found this for a previous online event: 
<img width="675" alt="Screenshot 2023-02-23 at 11 05 29" src="https://user-images.githubusercontent.com/122878451/220889767-709c57f7-8325-4e64-b99b-f595721c3c43.png">

What do you think @Jenya-design ?

**Comment by tadanna** [comment]
@Jenya-design awesome! I will just re-write these comments in that issue so that they don't get lost and discard this one. thanks!!

**Comment by Jenya-design** [comment]
Ok, thank you @tadanna 
I think we already have some events' templates in the themes. Please feel free to comment directly in the pdf files if necessary.
Your example can be easily made once we have the logo for a specific events. 
Similar template will be a bit empty as there's no logo now, but I can add it if you think it'll be useful
Let me know what you think

**Comment by tadanna** [comment]
@Jenya-design sorry I'm a little confused...I don't think we will create logos for every single event - we hope to have hundreds of them running! 
The logos should be for common/network events (like the summit). For general events I believe we should use the GSF logo, or we can discuss if we should create an "icon" (perhaps 1 for in-person and 1 for online). 

It could be on the lines of this, and the only thing we need to populate are the details of the specific event:

<img width="561" alt="Screenshot 2023-02-23 at 11 24 11" src="https://user-images.githubusercontent.com/122878451/220892968-97d39b8e-ad78-4f47-b832-bb72dc57f59f.png">

**Comment by Jenya-design** [comment]
@tadanna @holanita 
Is it possible for us to have a short call, so you can bring me up to date with how you're planning to structure the events?
Previously we had new event - new logo, new design. I see it'll change and I have to adjust the design accordingly
What will be the goups of events? Will you have some events under Summit group, some - general GSF events etc?
Sure, I can create the icons for for in-person and online events, no problem

**Comment by tadanna** [comment]
@Jenya-design sure let's plan a quick call. Can you drop on Slack your email and I'll see if I can schedule something for next week. 
I will try before our call to double check with Asim if he had already something in mind on how to structure this or if he's fine with what I thought. 
Also, this is not a priority at the moment but we will hopefully have the first events happening in March :)

**Comment by Jenya-design** [comment]
Thank you @tadanna 
I've sent you my email in slack

**Comment by jawache** [comment]
@tadanna / @Jenya-design - in case it helps this is what @tadanna is looking for, Hyperledger has an event cover template so every event looks this same, see: https://www.meetup.com/pro/hyperledger/

**Comment by Jenya-design** [comment]
Thank you @jawache 
Yes, it's useful and I understand in general what kind of item is required. Technically my main question for this particular task is what style and logo to use. General GSF style with only GSF logo on it? 

I asked for @tadanna to have a call with me (scheduled on Wednesday) as she mentioned that we won't make new logos for events anymore and the number of events will increase. Right now most of the templates I made suggest that there's a separate logo for the event. Like CarbonHack 22 by GSF. I would like to clarify this aspect and adjust the templates in the themes accordingly, so we have less issues with those in future.
Does that make sense?

**Comment by Jenya-design** [comment]
@tadanna I think we can create the issue for this upcoming event on the design board as [Update Design guidelines ](https://github.com/Green-Software-Foundation/ops-content/issues/6) has many items not related to it

You can put the list of the design items you need for the event - the checklist on the page 3 of [Theme 1]([Theme 1](https://drive.google.com/file/d/1FWr9DIdRrdMAzZxpgADpP7sZP1sAN3WH/view?usp=share_link)) might be helpful
I will start with the Meetup templates and continue with other items you'll need in March
Let me know what you think

cc @holanita

**Comment by tadanna** [comment]
@Jenya-design the CarbonHack is an umbrella event that has lots of events happening within different groups during the same time frame. What I am talking about is independent events that happen only once in a specific group. Therefore yes, I believe using the general GSF logo and style is what we need here. 
But let's discuss this briefly during our call on Wednesday, so I can explain it better and we can see how to set this up best :)

**Comment by Jenya-design** [comment]
@tadanna it's all clear for now, thank you again for the call
Please expect the update on Friday (I updated the deadline for this issue)

**Comment by Jenya-design** [comment]
Hello @tadanna 
Please check the [update for Meetup event cover](https://www.canva.com/design/DAFcFL_4gtw/izRhma7EJ8lL04tZEPjWtg/view?utm_content=DAFcFL_4gtw&utm_campaign=designshare&utm_medium=link&utm_source=publishsharelink)
There are 2 designs - with and without sponsors' logos in 3 color / illustration variations each
Let me know what you think
Thank you

**Comment by tadanna** [comment]
@Jenya-design I cannot see it - I have just requested the access permission :)

**Comment by Jenya-design** [comment]
Hi @tadanna apologies, I forgot that you're not in GSF Canva
Please check now, you should have the access to all GSF items
And I attach teh design as pdf here

[Meetup GSF event cover - 1200x676.pdf](https://github.com/Green-Software-Foundation/meetup/files/10882521/Meetup.GSF.event.cover.-.1200x676.pdf)

**Comment by Jenya-design** [comment]
Hi @jawache I think it's a missing step in onboarding
At some point Sean granted me the Administrator's rights so I could add the brang colors and elements. 
Since then I'm adding team members by request. I've added @tadanna to GSF canva team as soon as I saw her request, though I should check it earlier. 
I attach here the list of Canva members. Let me know if you need me to add / remove anybody or pass administrator's rights.

@tadanna  please let me know if you have any issues with the access

Thank you

![GSF Canva](https://user-images.githubusercontent.com/69306070/222770427-7455daa9-21bc-47ae-af12-6c51790cc883.png)

**Comment by jawache** [comment]
Oh very interesting! I think for now you should remain as the queen of canva and add/remove people as necessary. I'm not sure what the best process is for Canva, should just GSF staff have access or do we want to give our volunteers access like we have?

@tadanna and @Oleg-Zhymolokhov as the two newest starters can you maybe share notes and write up your thoughts on the onboarding process? And then let's use one of our team meeting to review and figure out both how to document it and how to streamline it.

**Comment by Jenya-design** [comment]
@jawache I cannot advise here as I'm not involved with GWs and volunteers. 
This is why I'm just sharing by request. It would be good to establish some process here

As Canva allows to edit the actual design I would prefer only GSF staff access. Though I assume we needed volunteers' help at least for one of the events. For now I'm just doing the backups before I share the design with GSF

**Comment by jawache** [comment]
I agree and lean more towards just having GSF members only with access, esp. if they have the ability to edit content. Feel free to remove anyone who's not a GSF staff member, if someone complains we'll know they were using it but I'm guessing no one is using it directly.

**Comment by Jenya-design** [comment]
Hello @jawache @zanete 
I've started removing some members from GSF Canva team, but I figured that they won't be able to access their designs anymore if they store the in GSF account (please check the screenshot). I'm not sure if the members have any relevant design items, but I hold this off just in case. I think Tammy and Jenny for example might have some pieces they used for the Summit.
I think it would be reasonable to ask the members if they have the backups of the files they might need.

@zanete can I ask you to help me to figure this out when you have time? I made you the administrator
Let me know what you think

![Image](https://user-images.githubusercontent.com/69306070/223780121-6928245d-fdb0-477d-b09d-561d2385f17a.png)

**Comment by Jenya-design** [comment]
Hello @jawache @tadanna 

I'm attaching here the preview for the Meetup event and speaker templates 
(requested by @jawache,  shared also via email)

Canva folder with editable templates for event here
https://www.canva.com/folder/FAFdEuD7gPA


![Image](https://user-images.githubusercontent.com/69306070/224692531-18b75afa-bf8a-40e2-bb76-1ab0535878e9.png)
![Image](https://user-images.githubusercontent.com/69306070/224692528-c0b03275-c805-47b1-879a-65785e91c8ac.png)
![Image](https://user-images.githubusercontent.com/69306070/224692530-7666537e-ec93-4ac6-86ac-e77f4b55e678.png)
![Image](https://user-images.githubusercontent.com/69306070/224692529-f2b0a828-3980-4e7d-b2b6-ded23b3a3987.png)

**Comment by tadanna** [comment]
hi @Jenya-design thanks for sharing all the assets, wonderful! I see that you created something a little different for the London meetup happening on March 23rd. If that is because you needed to include some elements that I didn't mention (very possible!) I believe we should move on with that or at least keep it within the templates so that it's not a one-off. 

Also, is there a reason why the folder in Canva is named "principles & patterns"?

thanks a lot again!

fyi @jawache

**Comment by Jenya-design** [comment]
Hello @holanita 
Please check the last version of the design attached
I created it based on the detailes that @jawache provided for the event in the _Re: March 24th 2023_ email thread and review by Sarah. The event is hosted by GSF and supported by Mastercard, there's no other sponsors, so I made both logos look more prominent and equal. Does that make sense?

I renamed the folder, let me know if it's consistent now. It was "principles & patterns" as I cionsidered it event's name from agenda.

Please let me know if you need any other design items for March 23rd event do by Thursday
I'll be away on Friday March 17, back to work on Moday midday
Thank you!

![GSF London Meetup March_event cover_1200x676-01](https://user-images.githubusercontent.com/69306070/224974307-43a89def-c419-4446-a3c9-8d56aa51f644.png)
![GSF London Meetup March_event cover_1200x676-02](https://user-images.githubusercontent.com/69306070/224974318-28c0c406-f7ef-468f-b0d9-a0410e258aab.png)
![GSF London Meetup March_speakers_1200x676](https://user-images.githubusercontent.com/69306070/224974319-83ead27c-7532-4aca-834f-0659e8151041.png)

**Comment by Jenya-design** [comment]
as requested by @NAMRATA-WOKE I've added options with _in-person and virtual_ for the event cover

Canva folder with editable templates for event here
https://www.canva.com/folder/FAFdEuD7gPA

![Image](https://user-images.githubusercontent.com/69306070/225300574-359bb93d-4dcf-45c1-9bef-bdef7a9a5c96.png)
![Image](https://user-images.githubusercontent.com/69306070/225300575-ba711f9d-7787-4e2e-96aa-d26ce1e9c7ea.png)

**Comment by Jenya-design** [comment]
Hi @tadanna @NAMRATA-WOKE 
In case you're working with this image, speackers' card is updated by Dan's request :
_Mastercard’s eagle-eyed branding team has spotted a capital C in Mastercard. They have asked if this can be changed please._


![Image](https://user-images.githubusercontent.com/69306070/225599426-419b8fa8-707a-4117-aec3-99e26a6965af.png)

**Comment by NAMRATA-WOKE** [comment]
Perfect, thank you!

**Comment by zanete** [comment]
@Jenya-design Apologies for leaving this ignored for so long

> @zanete can I ask you to help me to figure this out when you have time? I made you the administrator
Let me know what you think

I think it might be a bit of outside my scope to ask members if they need access. I would propose this: Remove access for anyone who's not in the GSF team, and then if anyone who has lost access tries to use them, they'd probably make a request again which can then be evaluated on a case by case basis. 

I'm not sure who would ultimately be in the position to decide or how they can be notified of this request (if all those notifications come to you @Jenya-design) but I feel that it would be quite evident in each case whether the person should or shouldn't have access. If you, for some reason, don't know the requester, hopefully you can reach the PM of the related project and can ask them (and use @jawache as fallback)

**Comment by Jenya-design** [comment]
Thank you @zanete 
I changed some transfer settings (it tooks 7 days to activate), that allowed me to transfer all the items from the deleted members to myself. I don't know exactly what those items are, as the person can copy and edit any shared designs, but no info should be lost in this case.

This is the list of persons, who now are in canva team now, let me know if I should remove anybody eslse
![image](https://user-images.githubusercontent.com/69306070/226947335-f963f6bb-bcd0-4656-ba79-3eef0c31e48e.png)

Thank you
cc @jawache

**Comment by tadanna** [comment]
@Jenya-design could you grant @Oleg-Zhymolokhov access to Canva? I will work with GSF only until the end of this week, and he will take over my tasks until the new meetup PM starts. 

@Oleg-Zhymolokhov if you are ever asked to create a cover image for the events from the organisers, you can do it yourself in Canva or if you have doubt ask Jenya!

**Comment by Jenya-design** [comment]
@Oleg-Zhymolokhov please give me your email I can use to send you the invitation

**Comment by Oleg-Zhymolokhov** [comment]
@Jenya-design it's oleg@greensoftware.foundation

**Comment by Jenya-design** [comment]
Thank you @Oleg-Zhymolokhov you should now have the invitation to GSF Canva
Here you can check Meetup and social media templates  https://www.canva.com/folder/FAFcFdS0HVQ
Please copy the design and save it in a sepatare folder before you create your version
Let me know if you have any questions or comments

@tadanna please also check [same folder](https://www.canva.com/folder/FAFcFdS0HVQ) when you have time and let me know if you have any comments or additional requests for these templates 
Thank you

**Comment by tadanna** [comment]
Thanks @Jenya-design I don't really have comments, I believe we will need to see how the design works for the next few events - aka: if we see that we constantly need to update the same elements, then we should update the template to what is the most common practice. 

@Oleg-Zhymolokhov please keep an eye on this (also for the new PM)!

**Comment by tadanna** [comment]
@Jenya-design could you also give access to Marc-Andre? His email is Marc-Andre@microsoft.com

@Oleg-Zhymolokhov I see now that Marc is not on GitHub. Once he has access to Canva could you please pick up the email conversation (RE: GSF Ottawa - Meetup Group Events 2023) and make sure he creates the cover image for the event? Might be that he will need help, so either you can Jenya can step in if needed now, but overall it would be greatif he becomes independent on this. 

This is another note for @jawache and @Oleg-Zhymolokhov : for organisers of member organisations I would give them access to Canva, for the ones that are non-members we should create the covers for them.

**Comment by Jenya-design** [comment]
@tadanna I've send the invitation for Marc and made you and @Oleg-Zhymolokhov admins, you should be able to add members now. Please remind the members to make a copy before they edit the design. Let me know if you need my help
Thank you

**Comment by Jenya-design** [comment]
@tadanna @Oleg-Zhymolokhov if you only need to share one or two templates, you can generate the link with the right to edit instead of inving the person to GSF team. Also you can delete members and transfer their assets to other team members
![image](https://user-images.githubusercontent.com/69306070/229095062-88882349-fe8a-4399-aa29-f1a35a3daa60.png)
![image](https://user-images.githubusercontent.com/69306070/229095164-71104f95-552d-4a14-8026-1b276d557a9a.png)

**Comment by Oleg-Zhymolokhov** [comment]
Hi Anna, Jenya!

Yes, I will pickup this conversation. Thank you for the info/access.

Oleg
